EIPS/eip-6404.md

```diff
@@ -0,0 +1,1000 @@
+---
+eip: 6404
+title: Transition to SSZ
+description: Unifies the CL and EL block representations
```

Comment:
This doesn't exactly describe what the EIP does, would rather see something like "Migrates to SSZ hash tree root commitment for transactions, receipts, and withdrawals".

Reply:
Reduced the scope to not include conversion of block hash, and went with more descriptive title